### PR TITLE
Remove last load/load_bytes methods

### PR DESCRIPTION
**Pull request by ret2libc**
I guess the time has come to remove load/load_bytes from RBinPlugin :)

Hopefully this will allow to clean even further the `r_bin_object_new` function and others as well in the future.

### CODE REVIEW

**Comment by radare** [comment]
so in total, the whole refactoring caused about -600LOC :D

File: libr/bin/p/bin_sms.c [hunk 1]
```diff
@@ -103,7 +103,7 @@ RBinPlugin r_bin_plugin_sms = {
 	.name = "sms",
 	.desc = "SEGA MasterSystem/GameGear",
 	.license = "LGPL3",
-	.load_bytes = &load_bytes,
+	.load_buffer = &load_buffer,
```
Comment (radare) [inline libr/bin/p/bin_sms.c]: the next step could be to do load_buffer -> load and check_buffer -> check :?

Comment (ret2libc) [reply libr/bin/p/bin_sms.c]: yeah it seems reasonable.

Comment (ret2libc) [reply libr/bin/p/bin_sms.c]: let's wait maybe a bit, just to not cause confusion

**Comment by radare** [comment]
Yeah we can wait one release for this breaking change so we can confuse api changes twice :D we have to break more apis \o/

**Pull request merged**